app/src/editor/canvas/index.jsx

```diff
@@ -22,7 +22,7 @@ import * as UndoContext from '$shared/contexts/Undo'
 import { Provider as PendingProvider } from '$shared/contexts/Pending'
 import Subscription from '$shared/components/Subscription'
 import * as SubscriptionStatus from '$shared/contexts/SubscriptionStatus'
-import { Provider as ClientProvider } from '$shared/contexts/StreamrClient'
+import ClientProvider from '$shared/contexts/StreamrClient'
```

Comment:
I would maybe prefer to have the `Provider` as a normal export, ie.

```js
import { Provider as ClientProvider } from '$shared/contexts/StreamrClient'
```

Reply:
Yep, it can be left as default but might be good to have the separate `Provider` export as well.

Reply:
Btw, exporting the raw ClientContext from `streamr-client-react` now (since v1.0.2).

Reply:
Sure, default export is fine also - my preference would be to have all exports named. But not a big deal, feel free to leave it.